README.md

```diff
@@ -43,8 +46,10 @@ flask run
 
 **Run flask windows**
 
-set FLASK_APP=flaskr
+cd backend
+
+$env:FLASK_APP= "flaskr"
```

Comment:
set FLASK_APP=flaskr will work on Windows.
The `$env:FLASK_APP="flaskr" is for Windows Power Shells. Maybe we can add both of them in?